From: Jiri Slaby <jslaby@suse.cz>
Subject: sysfs: spectre_v1 is mitigated
Patch-mainline: never, SUSE specific
References: bsc#1068032 CVE-2017-5753

$ cat /sys/devices/system/cpu/vulnerabilities/spectre_v1
Vulnerable

This is not correct as we have the BPF fix in place and we also have the
gmb/osb barriers engaged.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/x86/kernel/cpu/bugs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/kernel/cpu/bugs.c
+++ b/arch/x86/kernel/cpu/bugs.c
@@ -270,7 +270,7 @@ ssize_t cpu_show_spectre_v1(struct devic
 {
 	if (!boot_cpu_has_bug(X86_BUG_SPECTRE_V1))
 		return sprintf(buf, "Not affected\n");
-	return sprintf(buf, "Vulnerable\n");
+	return sprintf(buf, "Mitigation: barriers\n");
 }
 
 ssize_t cpu_show_spectre_v2(struct device *dev,
